webv2/README.md

```diff
@@ -0,0 +1,504 @@
+# HarbourBridge: Schema Assistant (alpha release)
+
+HarbourBridge is a stand-alone open source tool for Cloud Spanner evaluation and
+migration. This README provides details of the HarbourBridge schema assistant,
+which supports schema customization. For general HarbourBridge information see
+this [README](https:githubcomcloudspannerecosystemharbourbridge). Note that the
+schema assistant is currently in alpha release.
+
```

Comment:
Noticed the README, is the same as the older one. Please update and request review.

Reply:
Yes readme is not updated fully yet. We have added that in the backlog.

Reply:
Fixed the link.